CMakeLists.txt

```diff
@@ -157,27 +157,51 @@ endif()
 
 # Build the compare utilities
 if(ENABLE_ECL_INPUT)
-  add_library(testutil STATIC
-              examples/test_util/EclFile.cpp
-              examples/test_util/EclFilesComparator.cpp
-              examples/test_util/EclOutput.cpp
-              examples/test_util/EclRegressionTest.cpp
-              examples/test_util/EclUtil.cpp
-              examples/test_util/EGrid.cpp
-              examples/test_util/ERft.cpp
-              examples/test_util/ERst.cpp
-              examples/test_util/ESmry.cpp)
+  add_executable(compareECL
+    examples/test_util/EclFilesComparator.cpp
```

Comment:
When things are shuffled around anyways - maybe these should be moved out of the `examples/` directory - they are not examples after all.

Reply:
this is historical - the apps used to be in examples/. not protesting the change, just the rationale.

Reply:
I'd be happy to move them elsewhere if we can think of a good location. I'm a little reluctant however to add another top-level directory in opm-common since we already have `examples/`, `external/`, `jenkins/`, `msim/`, `opm/`, `redhat/`, `src/`, and `tests/`.